kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3Storage.java

```diff
@@ -60,14 +62,20 @@ public S3Storage(S3SinkConnectorConfig conf, String url) {
   }
 
   public AmazonS3 newS3Client(S3SinkConnectorConfig config) {
+    String proxyHost = config.getString(S3_PROXY_HOST);
+    int proxyPort = config.getInt(S3_PROXY_PORT);
+
+    ClientConfiguration clientConfiguration = PredefinedClientConfigurations.defaultConfig();
+    clientConfiguration.withUserAgentPrefix(String.format(VERSION_FORMAT, Version.getVersion()));
+    if(proxyHost != null && !proxyHost.isEmpty()) {
```

Comment:
System properties https.proxyHost and https.proxyport taken care in ClientConfiguration.java.
https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/ClientConfiguration.java#L602